shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -1241,4 +1241,11 @@ object GlutenConfig {
         + "partial aggregation may be early abandoned.")
       .intConf
       .createOptional
+
+  val COLUMNAR_VELOX_GLOG_VERBOSE_LEVEL =
+    buildConf("spark.gluten.sql.columnar.backend.velox.glogVerboseLevel")
+      .internal()
+      .doc("Set glog verbose level in Velox backend.")
```

Comment:
Can we add some details for this config? Mainly about how should the user set this config if wanting to change the level to INFO/WARN/ERROR etc.

Reply:
hmm.. I dont really want to add how to use glog in GlutenConfig.scala, user should search it by own.

this config does not control LOG(INFO) or LOG(WARNING), it control [verbose log](https://github.com/google/glog#verbose-logging), I will add another config to control INFO/WARNING/ERROR.

Reply:
https://github.com/oap-project/gluten/pull/3466

Reply:
I see. Maybe we can add sth. like below to let the user know how to adjust this value in Gluten.
`The lower the verbose level, the more likely messages are to be logged.`